tests/integration/features/conftest.py

```diff
@@ -359,6 +359,14 @@ def check_contents_plain(quteproc, text):
     assert text in content
 
 
+@bdd.then(bdd.parsers.parse('the page should not contain the plaintext '
+                            '"{text}"'))
+def check_contents_plain(quteproc, text):
```

Comment:
While it happens to work (because of the decorator doing magic), please name this function differently from the `check_contents_plain` above.